scripts/build/termux_step_get_dependencies.sh

```diff
@@ -111,12 +107,12 @@ termux_step_get_dependencies() {
 			fi
 			termux_run_build-package
 		fi
-	done<<<$(./scripts/buildorder.py $(test "${TERMUX_INSTALL_DEPS}" = "true" && echo "-i") "$TERMUX_PKG_BUILDER_DIR" $TERMUX_PACKAGES_DIRECTORIES || echo "ERROR")
+	done<<<$(./scripts/buildorder.py $([[ "${TERMUX_INSTALL_DEPS}" == "true" ]] && echo "-i") "$TERMUX_PKG_BUILDER_DIR" $TERMUX_PACKAGES_DIRECTORIES || echo "ERROR")
```

Comment:
```suggestion
	done< <(./scripts/buildorder.py $([[ "${TERMUX_INSTALL_DEPS}" == "true" ]] && echo "-i") "$TERMUX_PKG_BUILDER_DIR" $TERMUX_PACKAGES_DIRECTORIES || echo "ERROR")
```

Reply:
You will be surprised, but
```
$ time for ((i=0; i<10000; i++)); do read line < <(echo ""); done
real    0m16.229s
user    0m5.917s
sys     0m18.150s

$ time for ((i=0; i<10000; i++)); do read line <<< "$(echo "")"; done
real    0m14.410s
user    0m2.726s
sys     0m12.356s
```

Reply:
I don't really care if it's a couple milliseconds per run slower.
`< <()` is the idiomatically ***correct*** way to read input from a command.

`<<< $()` is valid, but sloppy.

Reply:
Ok, will be fixed in the next change batch.

Reply:
Done.